packages/wallet/frontend/src/utils/helpers.ts

```diff
@@ -55,10 +55,15 @@ export const formatAmount = (args: FormatAmountArgs): FormattedAmount => {
   }
 }
 
-export const formatDate = (date: string, time = true): string => {
+export const formatDate = (
+  date: string,
+  time = true,
+  month?: string
+): string => {
```

Comment:
```suggestion
export const formatDate = (
  date: string,
  time = true,
  month: Intl.DateTimeFormatOptions['month'] = 'short'
): string => {
```

Reply:
@Tymmmy Additionally, `formatDate` can accept only one argument - an object with `date`, `time` and `month`.